frontend/awx/main/routes/useAwxApplicationsRoutes.tsx

```diff
@@ -1,76 +1,76 @@
-import { useMemo, useState } from 'react';
-import { useTranslation } from 'react-i18next';
-import { PageNavigationItem } from '../../../../framework';
-import {
-  CreateApplication,
-  EditApplication,
-} from '../../administration/applications/ApplicationForm';
-import { ApplicationClientSecretModal } from '../../administration/applications/ApplicationPage/ApplicationClientSecretModal';
-import { ApplicationPage } from '../../administration/applications/ApplicationPage/ApplicationPage';
-import { ApplicationPageDetails } from '../../administration/applications/ApplicationPage/ApplicationPageDetails';
-import { ApplicationTokens } from '../../administration/applications/ApplicationPage/ApplicationPageTokens';
-import { Applications } from '../../administration/applications/Applications';
-import { Application } from '../../interfaces/Application';
-import { AwxRoute } from '../AwxRoutes';
+// import { useMemo, useState } from 'react';
```

Comment:
@keithjgrant do you want to keep this file around?

Reply:
thanks. deleted